src/edit/items/ItemForm.js

```diff
@@ -596,6 +596,8 @@ class ItemForm extends React.Component {
                           name="itemLevelCallNumberPrefix"
                           id="additem_callnumberprefix"
                           component={TextArea}
+                          format={v => v?.trim()}
+                          formatOnBlur
```

Comment:
What if a user saves via a shortcut?

Reply:
It also works with shortcuts

https://github.com/user-attachments/assets/51cccfff-c41c-4f30-8421-5ce2eb5554cb